mobile_robot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_bar_vslider
    pid: {p: 400.0, i: 200.0, d: 30.0}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_slider_slider
    pid: {p: 86.3819, i: 0.0, d: 40.0}